quickstart/README.md

```diff
@@ -26,17 +41,84 @@ kubectl get pods
 ```
 <!-- END bash -->
 
-TODO: more stuff with pod
+```
+NAME                        READY     STATUS    RESTARTS   AGE
+lobsters-3295177660-xg5t2   1/1       Running   0          55s
+```
+
+Check the log, use your specific pod name from above.
+
+```
+kubectl logs lobsters-3295177660-xg5t2
+```
+
+```
+[2016-04-07 17:36:37] INFO  WEBrick 1.3.1
+[2016-04-07 17:36:37] INFO  ruby 2.3.0 (2015-12-25) [x86_64-linux]
+[2016-04-07 17:36:37] INFO  WEBrick::HTTPServer#start: pid=1 port=3000
+```
+
+Let's see if lobesters is up and running, first we will do a private
+port forward from our machine to the container.
```

Comment:
I think it will, we will need additional instruction, there is a button on cloud shell to open a tab pointing to a port on the cloud shell machine itself.